<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0f326b;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #8b9bb5</string>
   </property>
   <property name="text">
    <string>CONTATOS</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGerarPdf">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #83a2d4; color:  #0f326b;</string>
   </property>
   <property name="text">
    <string>GERAR PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlterarContato">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #83a2d4; color:  #0f326b;</string>
   </property>
   <property name="text">
    <string>ALTERAR CONTATO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcluir">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #83a2d4; color:  #0f326b;</string>
   </property>
   <property name="text">
    <string>EXCLUIR CONTATO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVoltar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>650</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #83a2d4; color:  #0f326b;</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelaContatos">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>791</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #83a2d4; color:  #0f326b;;</string>
   </property>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
